### ISSUE TITLE
[Adaptive Cards> Show Card Wizard]: Validation is not set for the fields (Email and Number) present under show card wizard Cards.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9018]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: Dev (OS Build 27695.1000)
Browser Version: Version 130.0.2849.1 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro steps:**
1. Open the above URL in the edge dev browser.
2. Navigate to the 'Sample' option and press enter key to activate it.
3. Press tab key and navigate to the 'Sample Card Wizard' link and press enter key to activate it.
4. Press tab key and navigate the card and press tab key and navigate to 'Name' button.
5. Press enter key to expand it.
6. Now press tab key and navigate to the 'First Name' edit field enter some incorrect data.
7. Now press tab key and navigate to Address button and press enter key to expand it.
8. Fill the incorrect data in each field.
9. Press tab key and navigate to the 'Phone/Email' button and expand it.
10. Fill the incorrect data.
11. Press tab key and navigate to the Ok button and activate it using enter key and observe the issue.
12. 

**Actual Result:**
No validation is set for the Email and Phone Number fields present under Input card as after entering the incorrect value on activating the submit button no error message is getting displayed.

**Expected Result:**
Validation should be set for the fields (Email and Phone number) so that after entering the incorrect value, error message should be displayed.

**User Impact:**
It will impact the user as they will block to input the valid values due to this there might be chance to enter the incorrect values.

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Validation is not set for the fields (Address and Phone Email) present under show card wizard Cards](https://github.com/user-attachments/assets/1feaf930-0930-4a46-9a34-aa1980ffcbd6)

https://github.com/user-attachments/assets/d56b408c-701d-4fb5-83e4-a041c14e9e95

### Card JSON

```JSON
NA
```
